pr-data.csv

```diff
@@ -1165,12 +1165,18 @@ https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,extensi
 https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,extensions-core/kafka-indexing-service,org.apache.druid.indexing.kafka.supervisor.KafkaSupervisorTest.testReportWhenMultipleActiveTasks,ID,,,
 https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,extensions-core/kinesis-indexing-service,org.apache.druid.indexing.kinesis.supervisor.KinesisSupervisorTest.testIsTaskCurrent,ID,,,
 https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,extensions-core/kinesis-indexing-service,org.apache.druid.indexing.kinesis.supervisor.KinesisSupervisorTest.testShardSplit,ID,,,
-https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,core,org.apache.druid.java.util.emitter.core.EmitterTest.testBasicAuthenticationAndNewlineSeparating,ID,,,
-https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,core,org.apache.druid.java.util.emitter.core.EmitterTest.testBatchSplitting,ID,,,
-https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,core,org.apache.druid.java.util.emitter.core.EmitterTest.testGzipContentEncoding,ID,,,
-https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,core,org.apache.druid.java.util.emitter.core.EmitterTest.testSanity,ID,,,
-https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,core,org.apache.druid.java.util.emitter.core.EmitterTest.testSanityWithGeneralizedCreation,ID,,,
-https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,core,org.apache.druid.java.util.emitter.core.ParametrizedUriEmitterTest.testEmitterWithMultipleFeeds,ID,,,
+https://github.com/apache/druid,4b0d1b3488add75097532f25c8bbbcb35daa4987,processing,org.apache.druid.java.util.emitter.core.EmitterTest.testBasicAuthenticationAndNewlineSeparating,ID,,,
```

Comment:
Where are the logs showing that the new tests are `ID` in the commit SHA that you report? Please share info on Campuswire.